contributing/FORAMTTING-CODE.md

```diff
@@ -0,0 +1,169 @@
+# Formatting code blocks
+
+We want to adopt a consistent style for PowerShell code blocks and output in our documentation.
+The existing documentation has uses multiple styles and the formatting rules have changed multiple times.
+The current Open Publishing System (OPS) we use has a syntax highlighting feature that support many languages.
+OPS also adds a **Copy** button that copies the contents of the code block to the clipboard.
+This allows you to quickly paste the code into a script for testing the code example.
+However, not all examples in our documentation are intended to be run.
+Some code blocks are simple illustrations of a PowerShell concept.
+
+We are defining two scenarios for code blocks:
+
+1. Illustrative examples
+2. Executable examples
+
+## Formatting illustrative examples
+
+Illustrative examples are used to explain a PowerShell concept.
+They are not meant to be copied to the clipboard for execution.
+These are most commonly used for simple examples that are easy to type.
+They are also used for syntax examples where you are illustrating the syntax of a command.
+
+Illustrative examples use a "naked" code fence to mark the beginning and end of the code block.
+The code block may contain example output from the command being illustrated.
+
+### Syntax block
+
+Here is an example of a syntax block:
+
+    ```
+    Get-Command [-Verb <String[]>] [-Noun <String[]>] [-Module <String[]>]
+     [-FullyQualifiedModule <ModuleSpecification[]>] [-TotalCount <Int32>] [-Syntax] [-ShowCommandInfo]
+     [[-ArgumentList] <Object[]>] [-All] [-ListImported] [-ParameterName <String[]>]
+     [-ParameterType <PSTypeName[]>] [<CommonParameters>]
+    ```
+
+This is example is illustrating all of the possible parameters of the `Get-Command` cmdlet.
+
+Here is another syntax example that is describing the `for` statement in general terms:
+
+    ```
+    for (>init>; <condition>; <repeat>)
+    {<statement list>}
+    ```
+
+### Simple illustration example
+
+Here is an example illustrating PowerShell comparison operators:
+
+    ```
+    PS> 2 -eq 2
+    True
+
+    PS> 2 -eq 3
+    False
+
+    PS>  1,2,3 -eq 2
+    2
+
+    PS> "abc" -eq "abc"
+    True
+
+    PS> "abc" -eq "abc", "def"
+    False
+
+    PS> "abc", "def" -eq "abc"
+    abc
+    ```
+
+Note that this example has the simplified PowerShell prompt and shows the resulting output.
+In this case, we don't intend the reader to copy this example and try to run the copied code.
+
+## Formatting executable examples
+
+More complex examples or examples that would be useful to copy and execute should use <code>\`\`\`powershell</code> &hellip; <code>\`\`\`</code> code fence markers.
```

Comment:
Just an FYI, but I believe you can do this like:

```
`` ```powershell `` ... `` ``` ``
```
which will render as: 

`` ```powershell `` ... `` ``` ``

Source: https://gist.github.com/mcandre/8d76e076b2495fd8b36f439ec5033116

Reply:
This formatting was submitted by an external contributor and I accepted it then. The use of spaces gets dicey (especially if you get it wrong) and is harder to see the intent of the formatting.